pipelines/build/common/build_base_file.groovy

```diff
@@ -788,7 +788,7 @@ class Builder implements Serializable {
         def osArch = 'all available OS&ARCHs'
         def artifactsToCopy = '**/temurin/*.tar.gz,**/temurin/*.zip,**/temurin/*.sha256.txt,**/temurin/*.msi,**/temurin/*.pkg,**/temurin/*.json,**/temurin/*.sig'
         def dryRun = false 
-        def String releaseToolUrl = "${context.HUDSON_URL}job/build-scripts/job/release/job/refactor_openjdk_release_tool/parambuild?"
+        def String releaseToolUrl = "${context.JENKINS_URL}job/build-scripts/job/release/job/refactor_openjdk_release_tool/parambuild?"
```

Comment:
Please use `def` or `String`, but not both. See https://groovy-lang.org/style-guide.html#_def_and_type